static/styles/javascript/scripts.js

```diff
@@ -0,0 +1,98 @@
+$(document).ready(function() {
```

Comment:
I'm not really sure why this entire script is inside the document.ready call - most of this looks like it is dependent on being called by submitting the form so isn't required to be connected with the DOM on load

Reply:
Sorted. Didn't see I had JS and jQuery doc.ready.